build-managed.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 working_tree_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-$working_tree_root/run.sh build-managed $*
```

Comment:
I've been talking about this @MattGal 
The difference is is you pass in something like "aaa bb" The existing form would pass it as two parameters to calling script, "$@" preserves it as single parameter. I bumped into this while back when I was trying to pass multiple options to xunit 


https://stackoverflow.com/questions/12314451/accessing-bash-command-line-args-vs/12316565

I'm going to roll back this and submit that as separate PR.